101-streamanalytics-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "streamAnalyticsJobName": {
+      "value": "StreamAnalyticsJobName"
+    },
+    "numberOfStreamingUnits": {
```

Comment:
I will remove the default value, I prefer people stand still by this parameter because this is an important one (for performance and billing)